reference-implementation/lib/readable-stream.js

```diff
@@ -1151,6 +1147,17 @@ function ReadableStreamDefaultControllerHasBackpressure(controller) {
   return true;
 }
 
+// This is used in the implementation of TransformStream.
```

Comment:
Comment isn't in sync with the spec, since this is more widely used now.

Reply:
Thanks. Removed.